docs/APIGuide.md

```diff
@@ -39,8 +42,9 @@ Follow extension development guide https://github.com/SAP-samples/fsm-extension
         "hostingType": "KYMA"
     }
 ```
-5. Copy the extension id from the response body.
-6. Open POST **extensions deploy** API request. Replace {id} in the url with the copied extension id. Execute the request. Normally you do not need to enter request body, the extension will be installed into default namespace of KYMA. If you want to install into other namespace, enter it in request body like below:  
+5. Copy the id from the response body.
+6. Open POST **extensions deploy** API request. Replace {id} in the url with the copied extension id. In the Request body enter the namespace in that your Extension Installer in Kyma is set up.
```

Comment:
The namespace can be anyone, no need to keep the same as where extension installer lives

Reply:
Ok, than I misunderstood this. Thanks for clarification. Will update.

Reply:
added your sentence back in